suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -41,21 +58,26 @@ export const TransactionDetailHeader = ({
 }: TransactionDetailHeaderProps) => {
     const { FiatAmountFormatter } = useFormatters();
 
-    if (type !== 'recv' && type !== 'sent')
-        return <ErrorMessage errorMessage={`Unknown transaction type ${type}.`} />;
+    const { text } = transactionTypeInfo[type];
```

Comment:
```suggestion
    const { text, iconName, sign, signColor } = transactionTypeInfo[type];
```

Then you can omit these long object selectors as. `transactionTypeInfo[type].iconName` etc.

Reply:
They are only used in one place and are optional so this is better

Reply:
Why is it important that these are optional? 
```ts
const { iconName } = transactionTypeInfo[type];
```
and 
```tsx
const iconName  = transactionTypeInfo[type].iconName;
```
In case of the property is not defined, both expressions return exactly the same value, both are `undefined`. 

Don't you agree that 
```tsx
<Icon name={transactionTypeInfo[type].iconName} color="gray600" size="medium" />
```
looks better than:
```tsx
<Icon name={iconName} color="gray600" size="medium" />
```
?

Reply:
`const iconName  = transactionTypeInfo[type].iconName;`

is not the same as: 

`const { iconName }  = transactionTypeInfo[type];`.

If you try and apply your suggestion, typescript doesn't let you use it in that way, which makes sense, because if it's undefined, you shouldn't be able to. (viz screenshot)
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/36101761/215793735-bd0f413d-7ce2-4629-94c6-bc97236bf0c6.png">

Reply:
Don't you agree that

<Icon name={transactionTypeInfo[type].iconName} color="gray600" size="medium" />
looks better than:

<Icon name={iconName} color="gray600" size="medium" />


You mean it the other way around, as in the second one looks better, right?

Reply:
Ok, didn't know that Typescript disallows this. leave it like it is then.